spec/services/ccms/requestors/case_add_requestor_xml_blocks/non_passported_attributes_spec.rb

```diff
@@ -190,8 +190,8 @@ module Requestors
             end
           end
 
-          context "conditionally present land attributes" do
-            context "applicant has land" do
+          context "when conditionally present land attributes" do
```

Comment:
I don't think I understand what this line means

Reply:
Does
```suggestion
          context "when the conditionally present 'land' attribute is set" do
```
read any better @agoldstone93 ?

Reply:
Yep, makes sense. Thanks @colinbruce